src/lua/compat.c

```diff
@@ -7,9 +7,31 @@
 #include <lua/compat.h>
 
 static const struct luaL_Reg internal_compat[] = {
+	{"yaml_pretty_multiline_toggle", yaml_pretty_multiline_toggle},
 	{NULL, NULL},
 };
 
+static const char double_newline[] = "\n\n";
+static const char single_newline[] = "\n";
+
+char *FORCE_LITERAL_SUBSTRING = (char *)double_newline;
+
+int
+yaml_pretty_multiline_toggle(struct lua_State *L)
+{
+	assert(lua_isboolean(L, -1) && "boolean argument expected");
```

Comment:
Please use `assert(lua_isboolean(L, -1));` instead. I don't see the need for comments here.

Reply:
`assert()` prints where the error occurred, so no additional information is needed.

Reply:
Ok, removed it.